### ISSUE TITLE
concerns about training epochs

### ISSUE THREAD

**Issue by ForestsKing** [Secilia-Cxy/SOFTS#21]
Compared to iTransformer, I noticed that you increased the number of training epochs for Traffic. Based on my experimental results, under the same 10 epochs, there is no significant difference between SOFTS and iTransformer, and iTransformer has not yet fully converged at this point. Could you clarify whether a comparison under such experimental settings is truly fair?

**Comment by Halthward** [comment]
Have you successfully reproduced the performance metrics claimed in the paper on the ETTh2 dataset with the 336 and 720 step-length settings?

**Comment by Secilia-Cxy** [comment]
> Compared to iTransformer, I noticed that you increased the number of training epochs for Traffic. Based on my experimental results, under the same 10 epochs, there is no significant difference between SOFTS and iTransformer, and iTransformer has not yet fully converged at this point. Could you clarify whether a comparison under such experimental settings is truly fair?

Different methods require different hyperparameters. To ensure consistency with existing literature and avoid any discrepancies, we reported the iTransformer results directly from the values presented in the original paper. Since the authors would have chosen the most appropriate hyperparameters for their model, we believe this approach provides a fair comparison.

**Comment by Secilia-Cxy** [comment]
> Have you successfully reproduced the performance metrics claimed in the paper on the ETTh2 dataset with the 336 and 720 step-length settings?

The results might vary slightly due to differences in the PyTorch version used. Could you please try the following parameters and see if that helps?

python -u run.py \
  --is_training 1 \
  --root_path ./dataset/ETT-small/ \
  --data_path ETTh2.csv \
  --model_id ETTh2_96_336 \
  --model $model_name \
  --data ETTh2 \
  --features M \
  --seq_len 96 \
  --pred_len 336 \
  --e_layers 2 \
  --enc_in 7 \
  --dec_in 7 \
  --c_out 7 \
  --d_model 128 \
  --d_core 64 \
  --d_ff 128 \
  --learning_rate 0.0003 \
  --lradj cosine \
  --train_epochs 2 \
  --patience 3 \
  --des 'Exp' \
  --itr 1 \
  --batch_size 16 \
  --dropout 0.03

python -u run.py \
  --is_training 1 \
  --root_path ./dataset/ETT-small/ \
  --data_path ETTh2.csv \
  --model_id ETTh2_96_720 \
  --model $model_name \
  --data ETTh2 \
  --features M \
  --seq_len 96 \
  --pred_len 720 \
  --e_layers 2 \
  --enc_in 7 \
  --dec_in 7 \
  --c_out 7 \
  --d_model 512 \
  --d_core 256 \
  --d_ff 512 \
  --learning_rate 0.0003 \
  --lradj cosine \
  --train_epochs 20 \
  --patience 3 \
  --des 'Exp' \
  --itr 1
